package.json

```diff
@@ -38,24 +38,23 @@
     "date-fns-tz": "^2.0.0",
     "dom-to-image": "^2.6.0",
     "encoding": "^0.1.13",
-    "eslint": "8.47.0",
-    "eslint-config-next": "13.4.19",
-    "export-to-csv": "^1.2.1",
+    "export-to-csv": "^1.3.0",
     "fp-ts": "^2.16.5",
     "graphql": "^16.8.1",
     "graphql-request": "^6.1.0",
-    "i18n-iso-countries": "^7.7.0",
+    "i18n-iso-countries": "^7.11.1",
     "js-file-download": "^0.4.12",
-    "lucide-react": "^0.268.0",
-    "mapbox-gl": "^2.15.0",
+    "lucide-react": "^0.378.0",
+    "mapbox-gl": "^3.3.0",
     "next": "^13.4.19",
-    "postcss": "8.4.28",
-    "react": "18.2.0",
+    "postcss": "^8.4.38",
+    "react": "18.3.1",
     "react-day-picker": "^8.9.1",
-    "react-dom": "18.2.0",
-    "react-map-gl": "^7.1.6",
-    "recharts": "^2.12.3",
-    "sass": "^1.66.1",
+    "react-dom": "18.3.1",
+    "react-map-gl": "^7.1.7",
+    "recharts": "^2.12.6",
+    "sass": "^1.76.0",
+    "sharp": "^0.33.3",
```

Comment:
NextJS had a small warning that you could optimize images better if you used this library (`sharp`) and all you had to do was install it and NextJS would automatically start using it. Interesting I guess? Figured it was worth trying.